### ISSUE TITLE
WebXR Depth Sensing Module: What is DepthInfo.normDepthBufferFromNormView matrix? 

### ISSUE THREAD

**Issue by akssieg** [immersive-web/depth-sensing#37]
Inside [WebXR Depth Sensing Module](https://www.w3.org/TR/webxr-depth-sensing-1/) : 
What is the difference between "normalized view coordinates" and "depth buffer’s coordinate system" ? 
Does [normDepthBufferFromNormView](https://www.w3.org/TR/webxr-depth-sensing-1/#dom-xrdepthinformation-normdepthbufferfromnormview) provides the projection matrix to transform depth buffer’s array to normalized view coordinates?

**Comment by idrisshah** [comment]
Can we have a resolution for this change here? Thanks

**Comment by cabanier** [comment]
> Can we have a resolution for this change here? Thanks

Yes, I didn't hook this up to three.js and Quest browser always returns the identity matrix.
